Tools/i18n/.ruff.toml

```
@@ -0,0 +1,9 @@
+extend = "../../.ruff.toml"  # Inherit the project-wide settings
+target-version = "py313"
```

consistency with other `.ruff.toml`
```suggestion

target-version = "py313"
```